docker/daemon_unix.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/docker/docker/daemon"
 	"github.com/docker/docker/pkg/system"
 
-	_ "github.com/docker/docker/daemon/execdriver/lxc"
+	//"github.com/docker/docker/daemon/execdriver/lxc"
```

Comment:
Shouldn't have commented code.
Does this import need to go into a file with linux build tag?